### ISSUE TITLE
Fail to link with wxwidgets:wx64-windows-static

### ISSUE THREAD

**Issue by jsnhsu** [microsoft/vcpkg#13586]
Environment: Window 10, Visual Studio 2019 Community

I tried either '.\vcpkg.exe integrate install' or '.\vcpkg.exe integrate project' + 'Install-Package vcpkg.D.CppTool.vcpkg -Source "D:\vcpkg\scripts\buildsystems" ' don't work.  And always get the error like:

Error	LNK2019	unresolved external symbol __imp_InitCommonControls referenced in function "public: virtual bool __cdecl wxApp::Initialize(int &,wchar_t * *)" (?Initialize@wxApp@@UEAA_NAEAHPEAPEA_W@Z)	ConsoleApplication1	D:\<path>\ConsoleApplication1\wxmsw31ud_core.lib(app.cpp.obj)	1	

It looks like MSBuild can't find the lib under my project directory.(Q1)
I tried to copy the wxmsw31ud_core.lib to the path mentioned in above error.  It doesn't work either. (Q2)
It seems that MSBuild tried to look up the specifiied lib from current project directory (Q3, why?), even if I provide it, it still not able to see it.  How to build a MT project? (Q4)

My test code is very simple like:

#include <wx/frame.h>
class MyFrame : public wxFrame { };
int main()
{
    MyFrame f;
    f.Show();
	return 0;
}

Additional information: in the Property Page of this project, I set:

Use Vcpkg: Yes
Install vcpkg dependencies: Yes
Use static libraries: Yes
Use AutoLink: Yes
Triplet: $(VcpkgUserTriplet)
Vcpkg Configuration: $(Configuration)

**Comment by alek-d** [comment]
Looks like you need to link to `commctrl.lib`.

**Comment by jsnhsu** [comment]
It's a great hint. Thank you so much.  Although that one should be comctl32.lib and some other windows libs, like rprct14.lib, winmm.lib, advapi32.lib, etc...  Actually, the main() probably won't work here in my sample above.  So after took tens minutes to dig out one of the application I made with wxWidgets years ago.   I modified and create a class which inherits from wxApp and implements the OnInit() and OnExit().... now it works

So it's not an issue of vcpkg at all.
